### DISCUSSION TITLE
Does serverpod support GRPC?

### DISCUSSION THREAD

**Discussion by dinbtechit** [serverpod/serverpod#267] [Q&A] [answered]
Does serverpod support GRPC? if so can you point me to some examples. Thank you 🙏

**Comment by vlidholt** [comment, marked as answer]
Hi! Serverpod doesn't support gRPC, but instead uses its own protocol. The feature set is pretty much equivalent to gRPC though, but with some more automation (it generates the APIs from your backend code).

It's possible we will add a gRPC compatibility layer in the future.

**Comment by Wdestroier** [comment]
Does Serverpod encode small ints as variable-length integers too?
